internal/provisioner/plain/controllers/bundle_controller.go

```diff
@@ -455,6 +456,14 @@ func bundleGitRepoPod(pod *corev1.Pod, source rukpakv1alpha1.GitSource, unpackIm
 	return pod, nil
 }
 
+func (r *BundleReconciler) getCredential(ctx context.Context, secretname string) (string, string, error) {
+	secret, err := r.KubeClient.CoreV1().Secrets(r.PodNamespace).Get(ctx, secretname, metav1.GetOptions{})
```

Comment:
Should we use c-r dynamic client here? The only reason we have that kubernetes.Interface client present is because c-r doesn't have support for reading a Pod's log sub-resource.

Reply:
I see.  I probably remove this function.  I'll mount the secret instead of reading it.  Thanks!